src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateManager.cs

```diff
@@ -36,6 +36,8 @@ public static class CertificateManager
 
         public static EventHandler BridgeConfigurationChanged;
 
+        private const string CloudServerDomain = "cloudapp.net";
```

Comment:
We should not hard-code this. If there is no way to determine this dynamically, we should make it a TestProperty and BridgeConfiguration property.  Like BridgeDomain. Then we can pass it to build.cmd with a /p:BridgeDomain=cloudapp.net the same way we pass BridgeHost.